plugins/plugin-optimize/plugin.js

```diff
@@ -188,26 +118,65 @@ exports.default = function plugin(config, userDefinedOptions) {
         })
         .map((file) => path.join(buildDirectory, file)); // resolve to root dir
 
-      // 2. optimize all files in parallel
+      // 2. scan imports
+      const manifest = await scanHTML(
```

Comment:
🆕 Import manifest. This was needed because we first needed to see if any `.css.proxy.js` files were imported in the entire project before we injected a `<link />` tag in the HTML. This looks like…